mqttClient.js

```
@@ -0,0 +1,81 @@
+    let payload;
+
+    try {
+      payload = JSON.parse(message.toString());
+    } catch (e) {
+      logger.error("Invalid JSON in MQTT message");
+      return;
+    }
+
+    switch (topic) {
+      case 'avatar/emotion':
+        const emotion = payload.emotion;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)